cmd/proxy-to-grafana/proxy-to-grafana.go

```diff
@@ -46,6 +46,7 @@ var (
 	backendAddr  = flag.String("backend-addr", "", "Address of the Grafana server served over HTTP, in host:port format. Typically localhost:nnnn.")
 	tailscaleDir = flag.String("state-dir", "./", "Alternate directory to use for Tailscale state storage. If empty, a default is used.")
 	useHTTPS     = flag.Bool("use-https", false, "Serve over HTTPS via your *.ts.net subdomain if enabled in Tailscale admin.")
+	loginServer  = flag.String("login-server", "", "Address of alternative control server. If empty, the default Tailscale control is used.")
```

Comment:
I think you need to clarify that it's a URL, more than just an address: https://login.tailscale.com not login.tailscale.com

Reply:
In general I dont think we have done this, but I can change it to URL to control